mappings/net/minecraft/client/gui/screen/world/WorldEntryListWidget.mapping

```diff
@@ -1,72 +1,84 @@
-CLASS net/minecraft/unmapped/C_wwgrvynj net/minecraft/client/gui/screen/world/WorldListWidget
+CLASS net/minecraft/unmapped/C_wwgrvynj net/minecraft/client/gui/screen/world/WorldEntryListWidget
 	FIELD f_ahgrbbnb DATE_FORMAT Ljava/text/DateFormat;
-	FIELD f_biwkcpnf SNAPSHOT_FIRST_LINE Lnet/minecraft/unmapped/C_rdaqiwdt;
-	FIELD f_bnkwjzpz worldListFuture Ljava/util/concurrent/CompletableFuture;
+	FIELD f_biwkcpnf SNAPSHOT_TOOLTIP_FIRST_LINE Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_bnkwjzpz summariesListFuture Ljava/util/concurrent/CompletableFuture;
```

Comment:
```suggestion
	FIELD f_bnkwjzpz summaryListFuture Ljava/util/concurrent/CompletableFuture;
```
`summaryList` just has a nicer sound to it. looking at the code I don't see any reason to go with `summaries` instead!